packages/addons-website/scss/components/_swatch-palette-widget.scss

```diff
@@ -0,0 +1,89 @@
+@import '../prefix';
+
+.#{$prefix}--swatch-palettes {
+  display: flex;
+  flex-direction: column;
+  .#{$prefix}--swatch-palette {
```

Comment:
can you add a line break between 5 & 6, just a style rule we generally try and use 😄

Reply:
Will add a line break where ever nesting occurs